kiwrious/src/main/java/org/ahlab/kiwrious/android/KiwriousReader.java

```diff
@@ -12,41 +12,47 @@
 import org.ahlab.kiwrious.android.usb_serial.SerialService;
 import org.ahlab.kiwrious.android.utils.Constants;
 
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.ahlab.kiwrious.android.utils.Constants.KIWRIOUS_SERIAL_FRAME_SIZE_RX;
+import static org.ahlab.kiwrious.android.utils.Constants.supportedSensors;
+
 public class KiwriousReader {
     private static KiwriousReader instance;
 
-    private float conductivity = -1.2f;
-    private long resistance = -120;
-    private int voc = -32;
-    private int co2 = -32;
-    private int r = -255;
-    private int g = -245;
-    private int b = -235;
-    private float uv = -2.0f;
-    private long lux = -80;
-    private float humidity = -70;
-    private float temperature = -30;
-    private int ambientTemperature = -31;
-    private int infraredTemperature = -32;
-    private int heartRate = -72;
-
-    private byte[] rawValues;
-
-    private boolean isConductivityOnline = false;
-    private boolean isVocOnline = false;
-    private boolean isUvOnline = false;
-    private boolean isHumidityOnline = false;
-    private boolean isBodyTempOnline = false;
-    private boolean isHeartRateOnline = false;
-    private boolean isColorOnline = false;
+    private float conductivity = 0;
+    private long resistance = 0;
+    private int voc = 0;
+    private int co2 = 0;
+    private int r = 0;
+    private int g = 0;
+    private int b = 0;
+    private float uv = 0;
+    private long lux = 0;
+    private float humidity = 0;
+    private float temperature = 0;
+    private int ambientTemperature = 0;
+    private int infraredTemperature = 0;
+    private int heartRate = 0;
+
+    private byte[] rawValues = new byte[KIWRIOUS_SERIAL_FRAME_SIZE_RX];
 
     private SerialService serialService;
-    QueueWriter queueWriter;
-    QueueReader queueReader;
+
+    private QueueWriter queueWriter;
+    private QueueReader queueReader;
 
     public KiwriousCallback SensorConnected;
```

Comment:
Defined but never used. Currently this would be dead code since we don't yet use the callback function in our Usb Serial code. Maybe we hide this for now?